examples/key-export/src/pages/index.tsx

```diff
@@ -0,0 +1,213 @@
+import Image from "next/image";
+import styles from "./index.module.css";
+import { TurnkeyApiTypes } from "@turnkey/http";
+import { IframeStamper } from "@turnkey/iframe-stamper";
+import * as React from "react";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+type TPrivateKey = TurnkeyApiTypes["v1PrivateKey"];
+
+const TurnkeyIframeContainerId = "turnkey-iframe-container-id";
+const TurnkeyIframeElementId = "turnkey-iframe-element-id";
+
+export default function ExportPage() {
+  const [privateKeys, setPrivateKeys] = useState<TPrivateKey[]>([]);
+  const [iframeStamper, setIframeStamper] = useState<IframeStamper | null>(
+    null
+  );
+  const [showPrivateKey, setShowPrivateKey] = useState<boolean>(false);
+  const [selectedPrivateKey, setSelectedPrivateKey] = useState<string | null>(
+    null
+  );
+
+  useEffect(() => {
+    const instantiateIframeStamper = async () => {
+      const stamper = new IframeStamper({
+        iframeUrl: process.env.NEXT_PUBLIC_EXPORT_IFRAME_URL!,
+        iframeContainerId: TurnkeyIframeContainerId,
+        iframeElementId: TurnkeyIframeElementId,
+        iframeStyle: "border: none; width: 600px; height: 200px;",
```

Comment:
I'm wondering if this styling can be achieved by targeting the iframe element ID (instead of having a new `iframeStyle` property):

```css
#turnkey-iframe-element-id {
  border: none;
  width: 600px;
  height: 200px;
}
```

Reply:
i tried this a few different ways before having a new `iframeStyle` property and i might be missing something, but it didn't work :(

Reply:
wait a second... somehow i got something ELSE to work lol.... will push it in a second! but alas `iframeStyle` _can_ be removed after all

Reply:
```
.walletIframe iframe {
  border: none;
  width: 600px;
  height: 600px;
}
```
and
```
<div
  style={{ display: showWallet ? "block" : "none" }}
  id={TurnkeyIframeContainerId}
  className={styles.walletIframe}
/>
```